chef/cookbooks/crowbar/templates/default/crowbar-prepare-repositories.sh.erb

```diff
@@ -37,7 +40,7 @@ zypper --non-interactive addrepo -f <%= attrs[:url] %> <%= name %>
 zypper --non-interactive addrepo <%= attrs[:url] %> <%= name %>
     <% end -%>
 <% end %>
-zypper --non-interactive addrepo <%= @new_base_repo %> <%= @new_alias %>
+#zypper --non-interactive addrepo <%= @new_base_repo %> <%= @new_alias %>
```

Comment:
Could you just remove it? And also creation of `@new_base_repo` and `@new_alias` in the recipe..

Reply:
done but not tested yet